subprojects/plugin-development/src/main/java/org/gradle/plugin/devel/tasks/internal/ValidateAction.java

```diff
@@ -82,7 +85,8 @@ public void visitFile(FileVisitDetails fileDetails) {
                     try {
                         clazz = classLoader.loadClass(className);
                     } catch (IllegalAccessError | NoClassDefFoundError | VerifyError | ClassNotFoundException e) {
-                        throw new GradleException("Could not load class: " + className, e);
+                        LOGGER.debug("Could not load class: " + className, e);
```

Comment:
Would it make sense to distinguish between some of the errors here?

Reply:
Also, `DEBUG` sounds like a little too low, can we do `WARN` perhaps?

Reply:
I don't think `WARN` is helpful: a user would get tons of warnings for random classes which are not even tasks. Eventually if we move to bytecode analysis, there wouldn't be any failure/log any more.